benches/benches.rs

```diff
@@ -237,43 +352,105 @@ mod delayed_memory {
             self.m.clear().await
         }
     }
+
+    #[cfg(feature = "postgres-mem")]
+    impl<const ADDRESS_LENGTH: usize, const WORD_LENGTH: usize>
+        DelayedMemory<PostgresMemory<Address<ADDRESS_LENGTH>, [u8; WORD_LENGTH]>>
+    {
+        pub async fn clear(&self) -> Result<(), PostgresMemoryError> {
+            self.m.clear().await
+        }
+    }
 }
 
 fn bench_one_to_many(c: &mut Criterion) {
     #[cfg(feature = "redis-mem")]
-    let url = get_redis_url();
     let mut rng = CsRng::from_entropy();
 
-    #[cfg(feature = "redis-mem")]
+    #[cfg(any(feature = "redis-mem", feature = "postgres-mem"))]
     use delayed_memory::*;
 
     #[cfg(feature = "redis-mem")]
     bench_memory_one_to_many(
         "Redis",
         N_PTS,
-        async || DelayedMemory::new(RedisMemory::connect(&url).await.unwrap(), 1, 1),
+        async || DelayedMemory::new(RedisMemory::connect(&get_redis_url()).await.unwrap(), 1, 1),
```

Comment:
I personally preferred the old version since it fits in half my screen... but okay.